### ISSUE TITLE
CKV2_GCP_4 False positive

### ISSUE THREAD

**Issue by brettcurtis** [bridgecrewio/checkov-vscode#84]
**Describe the bug**

google_logging_project_sink fails on CKV2_GCP_4 if the bucket being used for logging sync has a variable set for retention_policy is_locked.

**To Reproduce**
You can at look this module: https://github.com/lzysh/terraform-google-project/blob/main/main.tf#L151 

This passes:
```hcl
  retention_policy {
    retention_period = 1800
    #is_locked        = var.bucket_lock
    is_locked        = true
  }
```

This fails:

```hcl
  retention_policy {
    retention_period = 1800
    is_locked        = var.bucket_lock
    #is_locked        = true
  }
```

**Expected behavior**
Both should pass since default variable is true.

**Additional context**
It also alerts on the follow resource:
![image](https://user-images.githubusercontent.com/1610100/147167477-4f5589c5-b738-4160-9627-7c2f7ae29290.png)

not the bucket resource, so that's a bit confusing.

This is a follow up on: https://github.com/bridgecrewio/checkov/issues/2142 since we are only able to reproduce this in vscode.
